src/optionalDependencies/JsCssPackage.web.ts

```diff
@@ -0,0 +1,13 @@
+let JsCssPackage: {postcss: any, cssjs: any} | undefined;
```

Comment:
I don't think you need two files for this package.. we can have only the:
`src/optionalDependencies/JsCssPackage.ts`
in the mobile we'll not use it anyway since the usage of this package is only from the web implementation of the SVG

Reply:
i know but i want to imply that by code.